functorch/functorch/_src/aot_autograd.py

```diff
@@ -740,7 +775,9 @@ def functional_call(*args, **kwargs):
             mod, pytree.tree_unflatten(args[:params_len], params_spec)
         ):
             if isinstance(mod, torch.fx.GraphModule):
-                with fx_traceback.override_stack_trace():
+                with fx_traceback.override_stack_trace(), torch.autograd.detect_anomaly(
```

Comment:
The docs on `DetectAnomalyGuard` indicate that this is a heavy func, only to be used for debugging and such.

Do we want to make this config driven? Or lighter weight?

Reply:
Stack tracing and propagation is only done once per compilation. 
In the actual execution, anomaly mode is not turned on.

Reply:
This adds a bunch of spam when benchmarking and gives users the misleading impression that we'll be slowing down execution:
```
/scratch/eellison/work/pytorch/functorch/functorch/_src/aot_autograd.py:815: UserWarning: Anomaly Detection has been enabled. This mode will increase the runtime and should only be enabled for debugging.
```

Can we turn off the warnings or suppress them if we're going to add this ?